lerna.json

```diff
@@ -1,5 +1,5 @@
 {
   "npmClient": "yarn",
   "useWorkspaces": true,
-  "version": "5.17.5"
+  "version": "independent"
```

Comment:
Because it's not the same for all packages.
`independent` is used in [material-ui](https://github.com/mui/material-ui/blob/master/lerna.json)
with `independent` running `yarn release:version` actually allows to bump the versions via cli correctly. 💪